src/javascript/binary/static/cashier/paymentagent_withdrawws.js

```diff
@@ -0,0 +1,271 @@
+var PaymentAgentWithdrawWS = (function() {
+    "use strict";
+
+    var containerID = '#paymentagent_withdrawal';
+    var $views      = $(containerID + ' .viewItem');
+    var errorClass = 'errorfield';
+    var viewIDs = {
+        error   : '#viewError',
+        success : '#viewSuccess',
+        confirm : '#viewConfirm',
+        form    : '#viewForm'
+    };
+    var fieldIDs = {
+        ddlAgents : '#ddlAgents',
+        txtAmount : '#txtAmount',
+        txtDesc   : '#txtDescription'
+    };
+    var formData, isValid;
+    var withdrawCurrency = 'USD',
+        minAmount = 10,
+        maxAmount = 2000;
+
```

Comment:
Sure, I hope so, but unfortunately there is no WS method returning them at the moment.
Please let me know if there was an API call which can be used to retrieve such values.